csharp/private/rules/Template.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>{FRAMEWORK}</TargetFrameworks>
+        <EnableDefaultEmbeddedResourceItems>False</EnableDefaultEmbeddedResourceItems>

```

Comment:
This is to prevent the default compilation includes of .NET projects. If this isn't specified, then any `.resx` file in the same directory as the generated `.csproj` will be included in compilation. I noticed this when testing with localized resource files (e.g. `Strings.resx`, `Strings.en-FR.resx`)

You can see more about this in .NET core projects [here](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#default-compilation-includes-in-net-core-projects) (I was using `dotnet` for testing this compilation)